lightning/src/ln/channel.rs

```diff
@@ -1230,12 +1228,17 @@ impl<Signer: Sign> Channel<Signer> {
 			holder_max_htlc_value_in_flight_msat: Self::get_holder_max_htlc_value_in_flight_msat(msg.funding_satoshis),
 			counterparty_selected_channel_reserve_satoshis: Some(msg.channel_reserve_satoshis),
 			holder_selected_channel_reserve_satoshis,
-			counterparty_htlc_minimum_msat: msg.htlc_minimum_msat,
 			holder_htlc_minimum_msat: if config.own_channel_config.our_htlc_minimum_msat == 0 { 1 } else { config.own_channel_config.our_htlc_minimum_msat },
 			counterparty_max_accepted_htlcs: msg.max_accepted_htlcs,
 			minimum_depth: Some(config.own_channel_config.minimum_depth),
 
-			counterparty_forwarding_info: None,
+			counterparty_forwarding_info: Some(CounterpartyForwardingInfo {
+				fee_base_msat: 0,
+				fee_proportional_millionths: 0,
+				cltv_expiry_delta: 0,
+				outbound_htlc_minimum_msat: Some(msg.htlc_minimum_msat),
+				outbound_htlc_maximum_msat: None,
+			}),
```

Comment:
As I mentioned in my comment above, I need to create a `CounterpartyForwardingInfo` object already here now, in order to be able to store the `msg.htlc_minimum_msat` value now that the  `Channel::counterparty_htlc_minimum_msat` has been removed. This forced me to set dummy values for `fee_base_msat`, `fee_proportional_millionths` and `cltv_expiry_delta` as we don't have the values for those yet. This isn't great... But making all of them optional values isn't that great either... Not sure what's best here.